src/workerd/api/util.c++

```diff
@@ -4,6 +4,7 @@
 
 #include "util.h"
 #include <kj/encoding.h>
+#include <unicode/uchar.h>
```

Comment:
You can remove the additional header now :-)

Reply:
Oh, yes! Done.